persist/fs/read.go

```diff
@@ -315,6 +324,16 @@ func (r *reader) ReadMetadata() (id ts.ID, length int, checksum uint32, err erro
 	return r.entryID(entry.ID), int(entry.Size), uint32(entry.Checksum), nil
 }
 
+func (r *reader) ReadBloomFilter() (managedConcurrentBloomFilter, error) {
```

Comment:
If this method is exported, we'll need to return an exported type.

Reply:
Its an unexported type with exported methods so people can still use it